src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -129,4 +132,25 @@
         @Config.Comment("Array of blacklisted dimension IDs in which Air Collector does not work.")
         public int[] airCollectorDimensionBlacklist = new int[]{};
     }
+
+    public static class NanoSaberConfiguration {
+
+        @Config.RangeDouble(min = 0, max = 100)
+        @Config.Comment("The additional damage added when the NanoSaber is powered. Default: 20f")
+        @Config.RequiresMcRestart
+        public double nanoSaberDamageBoost = 20;
+
+        @Config.RangeDouble(min = 0, max = 100)
+        @Config.Comment("The base damage of the NanoSaber. Default: 5f")
+        @Config.RequiresMcRestart
+        public double nanoSaberBaseDamage = 5;
+
+        @Config.Comment("Should Zombies spawn with NanoSabers? Default: true")
```

Comment:
I think this needs to mention in which case zombie can spawn with it.

Reply:
More clarification has been added.